Sources/Tracking/CustomerIOImplementation+ScreenViews.swift

```diff
@@ -24,18 +25,50 @@ extension CustomerIO {
         guard let swizzledMethod = class_getInstanceMethod(forClass, new) else { return }
         method_exchangeImplementations(originalMethod, swizzledMethod)
     }
+
+    func performScreenTracking(onViewController viewController: UIViewController) {
```

Comment:
I moved this function to make the code more testable in automated tests. Separating the `performAutomaticScreenTracking` logic with the logic that is given a UIViewController instance to track is what helped.